cmd/list.go

```diff
@@ -32,8 +32,15 @@ func newListCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	// listCmd represents the list command
 	var listCmd = &cobra.Command{
 		Use:   "list [repository]",
-		Short: "List the Appsody stacks available to init",
-		Long:  `This command lists all the stacks available in your repositories. If you omit the  optional [repository] parameter, the stacks for all the repositories are listed. If you specify the repository name [repository], only the stacks in that repository will be listed.`,
+		Short: "Lists the available Appsody stacks.",
+		Long: `Lists all the Appsody stacks available in your repositories. 
+
+An asterisk under the repository column denotes the default repository. An asterisk under the template column denotes the default stack template you will get when you initialise an Appsody project.`,
```

Comment:
Do we mean "under" the column, or "in"?
Not a fan of "you will get when" - how about " that is used when"?

Reply:
I have changed this to "in the column"